Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -92,14 +92,22 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
     .. group-tab:: minikube
 
-       Install minikube >= v1.5.2 as per minikube documentation: 
+       Install minikube >= v1.5.2 as per minikube documentation:
        `Install Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
 
        .. code-block:: shell-session
 
-          minikube start --network-plugin=cni
+          minikube start --network-plugin=cni --memory=4096
           minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
 
+.. note::
+
+   From minikube v1.12.1+, cilium networking plugin can be enabled directly with
+   ``--cni=cilium`` parameter in ``minikube start`` command. With this
+   flag enabled, ``minikube`` will not only mount eBPF file system but also
+   deploy ``quick-install.yaml`` automatically. However, this may not install the
+   latest version of cilium.
+
```

Comment:
I don't think we should have it, it's misleading and users might think it's an easier path to install Cilium, which it is, but installs an older Cilium version (1.8)

Reply:
I've already added a note about the version getting installed. It's useful to have a simpler step for users who don't care about version, and/or might miss the manual step.
Also, check this note from minikube command - 

>❗  With --network-plugin=cni, you will need to provide your own CNI. See --cni flag as a user-friendly alternative